<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>МЕНТОР II</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../r.qrc">
    <normaloff>:/images/apps/ooo-calc.png</normaloff>:/images/apps/ooo-calc.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="fraCentral">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>361</width>
      <height>241</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QFrame" name="fraTask">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </layout>
       </widget>
       <widget class="QFrame" name="fraQuiz">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="toolButton_12">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../r.qrc">
      <normaloff>:/ico/add.png</normaloff>:/ico/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_13">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>160</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../r.qrc">
      <normaloff>:/ico/add.png</normaloff>:/ico/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_14">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>160</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../r.qrc">
      <normaloff>:/ico/add.png</normaloff>:/ico/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_15">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>160</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset resource="../r.qrc">
      <normaloff>:/ico/add.png</normaloff>:/ico/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>787</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actNew"/>
    <addaction name="actOpen"/>
    <addaction name="actSave"/>
    <addaction name="actSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actTaskRoot"/>
    <addaction name="separator"/>
    <addaction name="actGenerate"/>
    <addaction name="actPrint"/>
    <addaction name="actGenerateForWeb"/>
    <addaction name="separator"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actHelpAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actNew"/>
   <addaction name="actOpen"/>
   <addaction name="actSave"/>
   <addaction name="actSaveAs"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actGenerate"/>
   <addaction name="actPrint"/>
   <addaction name="actGenerateForWeb"/>
   <addaction name="separator"/>
   <addaction name="actTaskRoot"/>
   <addaction name="separator"/>
   <addaction name="actSettingsGeneral"/>
  </widget>
  <action name="actNew">
   <property name="icon">
    <iconset resource="../r.qrc">
     <normaloff>:/ico/document-new.png</normaloff>:/ico/document-new.png</iconset>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="../r.qrc">
     <normaloff>:/ico/document-open.png</normaloff>:/ico/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actSave">
   <property name="icon">
    <iconset resource="../r.qrc">
     <normaloff>:/ico/document-save.png</normaloff>:/ico/document-save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actSaveAs">
   <property name="icon">
    <iconset resource="../r.qrc">
     <normaloff>:/ico/document-save-as.png</normaloff>:/ico/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="actGenerate">
   <property name="icon">
    <iconset resource="../r.qrc">
     <normaloff>:/ico/appointment-new.png</normaloff>:/ico/appointment-new.png</iconset>
   </property>
   <property name="text">
    <string>Варианты</string>
   </property>
   <property name="iconText">
    <string>Варианты</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actPrint">
   <property name="icon">
    <iconset resource="../r.qrc">
     <normaloff>:/ico/document-print.png</normaloff>:/ico/document-print.png</iconset>
   </property>
   <property name="text">
    <string>Печать вариантов</string>
   </property>
   <property name="iconText">
    <string>Печать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actSettingsGeneral">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../r.qrc">
     <normaloff>:/ico/document-properties.png</normaloff>:/ico/document-properties.png</iconset>
   </property>
   <property name="text">
    <string>Общие</string>
   </property>
  </action>
  <action name="actSettingsPrint">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../r.qrc">
     <normaloff>:/ico/document-print-preview.png</normaloff>:/ico/document-print-preview.png</iconset>
   </property>
   <property name="text">
    <string>Печать</string>
   </property>
  </action>
  <action name="actSettingsGeneration">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../r.qrc">
     <normaloff>:/ico/appointment-new.png</normaloff>:/ico/appointment-new.png</iconset>
   </property>
   <property name="text">
    <string>Генерация</string>
   </property>
  </action>
  <action name="actHelpContents">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../r.qrc">
     <normaloff>:/ico/edit-find.png</normaloff>:/ico/edit-find.png</iconset>
   </property>
   <property name="text">
    <string>Содержание</string>
   </property>
  </action>
  <action name="actHelpIndex">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Указатель</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actHelpAbout">
   <property name="icon">
    <iconset resource="../r.qrc">
     <normaloff>:/ico/gohome.png</normaloff>:/ico/gohome.png</iconset>
   </property>
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
  <action name="actTaskRoot">
   <property name="icon">
    <iconset resource="../r.qrc">
     <normaloff>:/ico/go-home.png</normaloff>:/ico/go-home.png</iconset>
   </property>
   <property name="text">
    <string>В начало</string>
   </property>
  </action>
  <action name="actGenerateForWeb">
   <property name="icon">
    <iconset resource="../r.qrc">
     <normaloff>:/ico/mail_new.png</normaloff>:/ico/mail_new.png</iconset>
   </property>
   <property name="text">
    <string>Генерация для web</string>
   </property>
   <property name="iconText">
    <string>Web-тесты</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../r.qrc"/>
 </resources>
 <connections/>
</ui>
